src/main/java/slimeknights/tconstruct/shared/block/SlimeBlock.java

```diff
@@ -30,22 +35,25 @@ public void fillItemGroup(ItemGroup group, NonNullList<ItemStack> items) {
   }
 
   public enum SlimeType implements IStringSerializable {
-    GREEN(0x01cd4e, 0x69bc5e),
-    BLUE(0x01cbcd, 0x74c5c8),
-    PURPLE(0xaf4cf6, 0xcc68ff),
-    BLOOD(0xb50101, 0xb80000),
-    MAGMA(0xff970d, 0xffab49),
-    PINK(0x90708b, 0xbc9eb4);
-
-    SlimeType(int color, int ballColor) {
+    GREEN(0x01cd4e, 0x69bc5e, Tags.Items.GREEN_SLIMEBALL),
+    BLUE(0x01cbcd, 0x74c5c8, Tags.Items.BLUE_SLIMEBALL),
+    PURPLE(0xaf4cf6, 0xcc68ff, Tags.Items.PURPLE_SLIMEBALL),
+    BLOOD(0xb50101, 0xb80000, Tags.Items.BLOOD_SLIMEBALL),
+    MAGMA(0xff970d, 0xffab49, Tags.Items.MAGMA_SLIMEBALL),
+    PINK(0x90708b, 0xbc9eb4, Tags.Items.PINK_SLIMEBALL);
+
+    SlimeType(int color, int ballColor, Tag<Item> slimeBall) {
       this.meta = this.ordinal();
       this.color = color;
       this.ballColor = ballColor;
+      this.slimeBallTag = slimeBall;
     }
 
     public final int meta;
     private final int color, ballColor;
+    private final Tag<Item> slimeBallTag;
     public static final SlimeType[] VISIBLE_COLORS = {GREEN, BLUE, PURPLE, BLOOD, MAGMA};
+    public static final SlimeType[] TINKER = {BLUE, PURPLE, BLOOD, MAGMA, PINK};
```

Comment:
Likewise, a JavaDoc on this array would not hurt. Explain what makes this different from `SlimeType.values()` and `SlimeType.VISIBLE_COLORS` (visible colors could use a javadoc too)

Reply:
well, I have no idea what VISIBLE_COLORS is being used for, idea says
its not used. And I have no idea what its supposed to be for, so can't really
document it.

Reply:
Visible colors is probably leftover from 1.12. In 1.12, we hid pink from creative tabs as its a fallback, not a main slime color. All the other types occur naturally. Feel free to leave no comment on it though, since its currently unused.